public_nba_data.Rmd

```diff
@@ -0,0 +1,56 @@
+# NBA data useful for a final project
+
+Dipesh Patel
+
+Sports is a common area for hobby data analysis and visualization. Here are some places to find publicly available data, for the NBA (National Basketball Association) in particular. Unique aspects of each data source are pointed out.
+
+### 1. stats.nba.com
```

Comment:
Other hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings. Correct: ## Subheading. Incorrect: ## 3. Subheading.